### ISSUE TITLE
Asking NYC Train Time for anything results in prompt to log in

### ISSUE THREAD

**Issue by someboredkiddo** [jprobinson/nyc-train-time#3]
Really great app, but something happened in the last few months where when I try and ask NYC Train Time anything I am always prompted for "Sorry, you need to be logged in for that to work..."

Guessing something with the OriginalDetectIntentRequest.Payload.User.UserID changed in dialogflow? See below for a Google Assistant Pixel 2 screenshot, but also happens with Google Minis

![image](https://user-images.githubusercontent.com/1919925/68050537-afcc5a80-fcbb-11e9-9a4b-34a5cf2fddb6.png)

**Comment by jprobinson** [comment]
Howdy @someboredkiddo! I'm unfortunately aware of this bug. A few months back, Google removed their anonymized user ID from the actions endpoint and some work is required to set up oauth + account linking to get things back.

I hope to have some time in the near future to do this. I'm guessing I'll have to pair the action with my similar website: https://wheresthetrain.nyc

**Comment by jprobinson** [comment]
Update on this one:

I've worked out the proper configurations and codes to deal with linking user accounts.

The changes are up for approval from Google. If all goes well, the action will be back in action within a week.

**Comment by jprobinson** [comment]
The release has been approved, NYC Train Time is back in action!

**Comment by someboredkiddo** [comment]
Thanks for taking the time to fix this! I was able to get the assistant commands back up and running and can go back to being slow to leave my apartment in the AM.

FYI Guessing Google did some tweaking with how intents / action dialogue is done and the `Ask NYC Train Time....` commands are confusing google assistant, but if I just get the app without a specific command the `MAIN` intent (I think?, guessing from the docs) lets me register my stop and then returns the time to my next train so I'm back in business

![image](https://user-images.githubusercontent.com/1919925/71275119-53340600-2322-11ea-821e-ce5e51b02d8a.png)

![image](https://user-images.githubusercontent.com/1919925/71275404-6941c680-2322-11ea-8584-02024219e41d.png)

**Comment by jprobinson** [comment]
Yeah, I ran into that problem as well. Any prompt to "Ask NYC Train Time..." causes Google to attempt to send an actual message/phone call to a contact with a similar name. It's a real bummer.

The only way I've been able to get things to work so far is to use the same prompt "Talk to NYC Train Time" but immediately mention another train/station I care about. For example saying "Talk to NYC Train Time 6 train" should reply asking for a specific stop.
